repo/content/content_manager_lock_free.go

```diff
@@ -23,8 +23,7 @@ const indexBlobCompactionWarningThreshold = 100
 
 // lockFreeManager contains parts of Manager state that can be accessed without locking
 type lockFreeManager struct {
-	// this one is not lock-free
-	Stats Stats
+	Stats *Stats
```

Comment:
The reason this is being converted into a pointer is because when `lockFreeManager` is embedded into `content.Manager` then `Stats` is no longer aligned, is that correct?

The downside of making it a pointer is that it could now be `nil`. In this case, it's probably OK since this is only instantiated in `content.Manager` and the _right thing happens_.